eo_protocol.txt

```diff
@@ -1,94 +1,105 @@
 
 enum PacketFamily : byte
 {
-	 1 Connection
+	 1 Security
```

Comment:
As I said on discord I have mixed feelings about the rename. On the one hand the updates are more correct, but people have known these packets by the eoserv names for 15 years. These names _do_ make a lot of sense, and I think people would eventually get used to them, but it could be kind of confusing.

Maybe it wouldn't be a bad idea to keep a branch around that can take fixes for the v28 spec, and maintain the new updates separately?

Reply:
Going to have a good think about this.. I really do want a pure v28 protocol definition and then basically "forks" for the new stuff. Maybe the answer is we just have folders in the root

```
/protocol/v28/..
/protocol/v28-eoserv-names/..
/protocol/v29/..
```